topics/templates/document-attributes-product.adoc

```diff
@@ -51,6 +51,7 @@
 :apidocs_link: {project_doc_base_url}/api_documentation/index
 :developerguide_name: Server Developer Guide
 :developerguide_link: {project_doc_base_url}/server_developer_guide/
+:developerguide_deploying_themes: {developerguide_link}#deploying-themes
```

Comment:
The anchor doesn't work, should be an underscore, apparently:

https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_developer_guide/index#deploying-themes

vs 

https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_developer_guide/index#deploying_themes

Reply:
This is interesting as the underscore doesn't work for the upstream: https://www.keycloak.org/docs/latest/server_development/index.html#deploying-themes

Reply:
You're absolutely right, @ASzc! Fixing.

Reply:
Done.